packages/utils/src/lib/verbose-utils.ts

```diff
@@ -1,20 +1,20 @@
-function getLogVerbose(verbose: boolean) {
+function getLogVerbose(verbose = false) {
```

Comment:
Now that the verbose functions have a default, there should be at least one test calling the function without an argument to cover this use case.

Reply:
added test